recipes/python/package.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+@early()
+def name():
+    import os
+    name = os.path.basename(os.path.dirname(os.getcwd()))
+    return name
+
+@early()
+def version():
+    import os
+    version = os.path.basename(os.getcwd())
+    return version
+
+@early()
+def uuid():
+    import uuid
+    return str(uuid.uuid5(uuid.NAMESPACE_DNS, name()))
+
+description = \
+    """
+    An interpreted, interactive, object-oriented programming language
+    """
+
+vendor_name = "Python Software Foundation"
+
+vendor_type = "external"
+
+opensource = True
+
+source_author = vendor_name
+
+source_license = "PSF"
+
+source_type = "c++"
+
+gpl_compatible = True
+
+has_plugins = False
+
+vendor_version = version()
+
+build_type = "python"
+
+package_author = "maxnbk"
+build_author = "maxnbk"
+
+relocateable = False
+
+build_requires_source = True
+build_requires_internet = False
+build_requires_sudo = False
```

Comment:
might wanna drop these from the installed pkg def, prefix with `__` to do that, as per https://github.com/nerdvegas/rez/wiki/Package-Definition-Guide#package-attributes

Reply:
I'd think people might want to rez-search these attributes in the future in their installed packages, so as to be able to identify builds that use sudo, for example, and try to fix them to make them not require it.